pkg/config/parameters.go

```diff
@@ -421,6 +453,10 @@ func (p *Parameters) Validate() error {
 		return err
 	}
 
+	if err := p.GatewayConfig.Validate(); err != nil {
```

Comment:
Should we only do validation of these bits if the service APIs CLI flag is set?

Reply:
That flag will go away, so the idea here is to validate what the user inputed. Both are required according to Nick's design, but will default if not specified.